$NetBSD: patch-am,v 1.6 2012/07/05 10:23:31 wiz Exp $

First chunk: Fix build with poppler-0.20.
Second chunk: Fix build with png-1.5.

--- src/extension/internal/pdfinput/svg-builder.cpp.orig	2011-07-08 18:25:09.000000000 +0000
+++ src/extension/internal/pdfinput/svg-builder.cpp
@@ -961,9 +961,7 @@ void SvgBuilder::updateFont(GfxState *st
     _font_style = sp_repr_css_attr_new();
     GfxFont *font = state->getFont();
     // Store original name
-    if (font->getOrigName()) {
-        _font_specification = font->getOrigName()->getCString();
-    } else if (font->getName()) {
+    if (font->getName()) {
         _font_specification = font->getName()->getCString();
     } else {
         _font_specification = (char*) "Arial";
@@ -1443,7 +1441,7 @@ Inkscape::XML::Node *SvgBuilder::_create
         return NULL;
     }
     // Set error handler
-    if (setjmp(png_ptr->jmpbuf)) {
+    if (setjmp(png_jmpbuf(png_ptr))) {
         png_destroy_write_struct(&png_ptr, &info_ptr);
         return NULL;
     }
